source/sql/alembic/versions/56c1c7a19078_add_visible_parameter.py

```diff
@@ -0,0 +1,25 @@
+"""add visible parameter
+
+Revision ID: 56c1c7a19078
+Revises: 46168c7abc89
+Create Date: 2018-11-13 15:33:38.146755
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '56c1c7a19078'
+down_revision = '46168c7abc89'
+
+from alembic import op
+import sqlalchemy as sa
+import geoalchemy2 as ga
+from sqlalchemy.dialects import postgresql
+
+
+def upgrade():
+    ### Add new flag for input/ouput visibility
+    op.add_column('way', sa.Column('visible', sa.BOOLEAN(), nullable=False), schema='georef')
```

Comment:
You need to add te default value:
op.add_column('way', sa.Column('visible', sa.BOOLEAN(), nullable=False, server_default='false'), schema='georef', )

Reply:
Actually, the autogenerate command wrote this.
I not a expert with alembic. So, I do what you want :gift: 

I'll retest the migration to be sure

Reply:
The default value of visible should be `server_default='true'` here most probably.

Reply:
op.add_column('way', sa.Column('visible', sa.BOOLEAN(), nullable=False, server_default='true'), schema='georef') works well

Reply:
After krishna's test, with server_default='true' it is OK